pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -3537,103 +3538,6 @@ spec:
 	}
 }
 
-// TestReconcileWithTimeoutAndRetry runs "Reconcile" against pipelines with
-// retries and timeout settings, and status that represents different number of
-// retries already performed.  It verifies the reconciled status and events
-// generated
-func TestReconcileWithTimeoutAndRetry(t *testing.T) {
```

Comment:
can we modify this test instead of removing it completely?

Reply:
discussed offline with @XinruZhang - this test is testing how the retriesStatus is updated which is not of relevance to pipelinerun reconciler anymore